src/rollup/widgets/close-widget.js

```diff
@@ -0,0 +1,49 @@
+/** @module pathfora/widgets/close-widget */
+
+import document from '../dom/document';
+
+import trackWidgetAction from '../data/tracking/track-widget-action';
+
+import removeClass from '../utils/class/remove-class';
+import hasClass from '../utils/class/has-class';
+
+import { widgetTracker } from '../globals/config';
+
+
+export default function closeWidget (id, noTrack) {
+  var node = document.getElementById(id);
+  var i;
```

Comment:
nit but seems like in most places we put undefined variables at the top level of the var definition and separate additional definitions with commas rather than multiple var statements. not sure i have a preference, both have benefits and drawbacks, but we should at least be consistent. nit tho for sure.